src/app.ts

```diff
@@ -115,6 +115,7 @@ export function generateServerlessRouter(
                 ...requestInformation,
                 requestContext: res.locals.requestContext,
                 accessToken: req.headers.authorization,
+                fhirServiceBaseUrl: res.locals.serverUrl,
```

Comment:
This was probably set in a previous PR but where is `res.locals.serverUrl` initialized?

Reply:
Here:
https://github.com/awslabs/fhir-works-on-aws-routing/blob/552a1180697a6406950c4a27e6b0cc0b3a0b2f76/src/router/middlewares/setServerUrl.ts#L12-L16

Reply:
Thanks